megamek/src/megamek/client/bot/BotClient.java

```diff
@@ -730,7 +730,7 @@ protected List<Coords> getStartingCoordsArray(Entity deployed_ent) {
         }
         for (Mounted<?> mounted : deployed_ent.getAmmo()) {
             AmmoType ammoType = (AmmoType) mounted.getType();
-            if (ammoType.getAmmoType() == AmmoType.T_ATM) {
+            if (ammoType.getAmmoType() == AmmoType.AmmoTypeEnum.ATM) {
```

Comment:
Your IDE may be able to strip at least `AmmoType.AmmoTypeEnum` here; it's not much, but it reduces some of the boilerplate.
Not a requirement, just letting you know that's an option if you weren't already aware.